test/new_relic/telemetry_sdk_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class NewRelic::TelemetrySdkTest < Minitest::Test
+  def test_that_it_has_a_version_number
+    refute_nil ::NewRelic::TelemetrySdk::VERSION
```

Comment:
Do we want to supply a convenience method for returning VERSION as a GemVersion instance?

https://ruby-doc.org/stdlib-2.5.0/libdoc/rubygems/rdoc/Gem/Version.html